docker-compose.yaml

```diff
@@ -72,7 +72,7 @@ services:
   
     suricata:
         container_name: pcapmonkey_suricata
-        image: jasonish/suricata:6.0.0
+        image: jasonish/suricata:latest
```

Comment:
I think we should keep pinning the container to a known release. It's much safer this way, and we should manually testing the suricata versione before updating it